### PR TITLE
refactor: improve navbar, footer, awards, search engine, fix missing cdl

### PR DESCRIPTION
**Pull request by drendog**

### CODE REVIEW

File: src/app/coccarda/Coccarda.tsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
+import { faAward } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import React from 'react';
-import './Coccarda.scss';
 
 function Coccarda(): JSX.Element {
   return (
-    <img src="coccarda.png" alt="eletto" className="m-2 coccarda" />
+    <FontAwesomeIcon className="mx-2" icon={faAward} color="red" size="lg" />
```
Comment (Helias) [inline src/app/coccarda/Coccarda.tsx:7]: nice :+1:

**Comment by drendog** [comment]
[live version](https://drendog.github.io/UNICT-Elezioni/)

**Comment by Helias** [comment]
I prefer to have these text in white
image

**Comment by Helias** [comment]
Increase the padding of the footer please (and/or decrease the font-size of the text)

image

File: src/app/search/SearchEngine.ts [hunk 2]
```diff
@@ -331,10 +338,48 @@ class SearchDepartment {
...
 
+class SearchCdl {
+  search(str: string, limits: SearchLimits): CdlInfo[] {
+    const results: CdlInfo[] = [];
+    str = str.replace(' ', '_').toUpperCase();
+    for (const year of years) {
+      for (const cdl of (cdls as any)[year]) {
+        if (limits.isFull()) {
+          return results;
+        }
```
Comment (Helias) [inline src/app/search/SearchEngine.ts:349]: you can do this at the beginning of the function

File: src/app/search/SearchEngine.ts [hunk 3]
```diff
@@ -331,10 +338,48 @@ class SearchDepartment {
...
+            isUnder500: false
+          });
+          limits.increaseResults();
+        }
+      }
+
+      for (const cdl of (cdl500 as any)[year]) {
+        if (limits.isFull()) {
+          return results;
+        }
```
Comment (Helias) [inline src/app/search/SearchEngine.ts:363]: ... and save this code

File: src/app/search/SearchEngine.ts [hunk 4]
```diff
@@ -331,10 +338,48 @@ class SearchDepartment {
...
+    }
+    return results;
+  }
+}
+
 class SearchEngine {
   private searchDep = new SearchDepartment()
   private searchList = new SearchList()
   private searchCandidate = new SearchCandidate()
+  private searchCdl = new SearchCdl()
```
Comment (Helias) [inline src/app/search/SearchEngine.ts:382]: `;`

File: src/app/search/SearchForm.tsx [hunk 5]
```diff
@@ -31,21 +38,61 @@ export const SearchForm = (): JSX.Element => {
     ) as any;
   }
 
+  function getCdlUrl(suggestion: CdlInfo) : string {
```
Comment (Helias) [inline src/app/search/SearchForm.tsx:41]:
```suggestion
  function getCdlUrl(suggestion: CdlInfo): string {
```

File: src/app/search/SearchForm.tsx [hunk 6]
```diff
@@ -31,21 +38,61 @@ export const SearchForm = (): JSX.Element => {
...
+                  <FontAwesomeIcon icon={faGraduationCap} size="4x"></FontAwesomeIcon>
+                </td>
+                <td className="col">{suggestion.name.replaceAll('_', ' ')}</td>
+                <td className="col text-nowrap">{suggestion.year}</td>
+              </tr>
+            </tbody>
+          </table>
+        </ListGroup.Item>
+      </a>
+    ) as any;
```
Comment (Helias) [inline src/app/search/SearchForm.tsx:65]: I am not sure about the `as any`

File: src/app/search/SearchForm.tsx [hunk 7]
```diff
@@ -73,38 +133,49 @@ export const SearchForm = (): JSX.Element => {
...
+            <InputGroup.Append>
+              <Button type="submit" variant="primary"><FontAwesomeIcon icon={faSearch} /></Button>
+            </InputGroup.Append>
+          </InputGroup>
+        </Form>
+        <ListGroup className={formValue.length ? 'suggestions-list' : 'd-none'}>
+          {generateSuggestions()}
+        </ListGroup>
+      </div>
+      <div className="page-overlay" onClick={(): void => props.onClose()} />
```
Comment (Helias) [inline src/app/search/SearchForm.tsx:178]: :thinking:

**Pull request merged**